main.js

```diff
@@ -1,3 +1,28 @@
 import './style.css';
 
-console.log('Hello world');
+let btn2 = document.getElementById("tooltip-2");
+let tooltip2 = document.getElementById("tooltip-content-2");
+let btn3 = document.getElementById("modal-button");
+let modal = document.getElementById("modal-window");
+let shadow = document.getElementById("shadow");
+let modalCloseButton = document.getElementById("modal-close-button");
+
+function updateTooltip2(event) {
+	console.log(event.target);
+  if (event.target == btn2 && tooltip2.style.display != "block") {
+    tooltip2.style.display = "block";
+  } else {
+    tooltip2.style.display = "none";
+  }
+  if (event.target == btn3 && modal.style.display != "block") {
```

Comment:
Place the `<span>` inside this button and check the result. I have talked about that on lecture.